rust/pkg/cardano_serialization_lib.js.flow

```diff
@@ -6252,11 +6352,46 @@ declare export class TransactionOutput {
    */
   data_hash(): DataHash | void;
 
+  /**
+   * @returns {PlutusData | void}
+   */
+  data(): PlutusData | void;
+
+  /**
+   * @returns {ScriptRef | void}
+   */
+  script_ref(): ScriptRef | void;
+
+  /**
+   * @param {ScriptRef} script_ref
+   */
+  set_script_ref(script_ref: ScriptRef): void;
+
+  /**
+   * @param {PlutusData} data
+   */
+  set_data(data: PlutusData): void;
+
   /**
    * @param {DataHash} data_hash
    */
   set_data_hash(data_hash: DataHash): void;
 
+  /**
+   * @returns {boolean}
+   */
+  has_plutus_data(): boolean;
```

Comment:
Let's rename the setter and the getter into `.set_plutus_data` and `.plutus_data()` then, cuz just generic "data" is too vague, as demonstrated by this boolean function